tests/h/services/search_index/service_test.py

```diff
@@ -306,3 +298,18 @@ def search_index(es_client, pyramid_request, settings_service, queue):
 @pytest.fixture(autouse=True)
 def es_client():
     return create_autospec(Client, instance=True)
+
+
+@pytest.fixture(autouse=True)
+def indexer(patch):
+    return patch("h.services.search_index.service.indexer")
+
+
+@pytest.fixture(autouse=True)
+def report_exception(patch):
+    return patch("h.services.search_index.service.report_exception")
```

Comment:
`report_exception()` was being used in `service.py` but wasn't patched, which meant that `service_test.py` was executing `h_pyramid_sentry` and `sentry_sdk` code. I don't think `sentry_sdk` tries to access the network as long as the `SENTRY_*` envvars aren't set in the tests. But still we don't want to be calling this code, this import should be patched. (Normally we _wouldn't_ patch more third-party imports but I think we do want to patch `h_pyramid_sentry` / `sentry_sdk`.)